### ISSUE TITLE
PoseAction velocity is always Zero

### ISSUE THREAD

**Issue by dpggit** [ValveSoftware/steamvr_unity_plugin#603]
I am trying to use the poseAction[handType].velocity and always give me Zero, GetTrackedObjectVelocity worked in previous version in 2.5 gives zero vector3

**Comment by dpggit** [comment]
Please make updates to improve things not to make it worse, i am getting a headache with steamvr

**Comment by zite** [comment]
Comments like this are not helpful and will not get us to answer your questions faster

**Comment by zite** [comment]
It sounds like you are also using Vive Input Utility which may be causing the issues you are seeing. Can you verify you see this behavior in a new project?

**Comment by dpggit** [comment]
No, i am not using VIU anymore, anyways i just made my own method to detect velocity
